frontend/ViewModel/MainViewModel.cs

```diff
@@ -113,5 +132,34 @@ private void ConnectWs()
         {
             _messenger.Connect();
         }
+
+        private void SignIn(object parameter)
+        {
+            //if authUser() == true 
+            SetConfigData(CurrentUser.Login, (parameter as PasswordBox).Password);
+            RestartApp();
+          
+        }
+
+        private void SignOut()
+        {
+            SetConfigData();
+            RestartApp();
+        }
+
+        private void SetConfigData(string login="", string password="")
+        {
+            Configuration config = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
+            config.AppSettings.Settings["login"].Value = login;
+            config.AppSettings.Settings["password"].Value = password;
+            config.Save(ConfigurationSaveMode.Modified);
+            ConfigurationManager.RefreshSection("appSettings");
+        }
+
+        private void RestartApp()
+        {
+            System.Windows.Forms.Application.Restart();
```

Comment:
do you really need a windows forms for this?
( not sure but probably Application is accesible from wpf enviroment )